src/components/post-card/PostCard.jsx

```diff
@@ -1,15 +1,24 @@
 import "./PostCard.css";
-import { avatar, Icon_delete } from "../../assets";
+import { avatar } from "../../assets";
 import { useState } from "react";
 import { Link } from "react-router-dom";
 import { useAuthContext } from "../../context/AuthContext";
 import { PostApi } from "../../apicalls/PostApi";
 import { useAppContext } from "../../context/AppContext";
```

Comment:
Suggestion: these context files can be import from one index.js file from context folder

Reply:
Done